packages/google_maps_flutter/google_maps_flutter_android/lib/src/google_maps_flutter_android.dart

```diff
@@ -1146,6 +1146,22 @@ PlatformZoomRange? _platformZoomRangeFromMinMaxZoomPreferenceJson(
   return PlatformZoomRange(min: minMaxZoom[0], max: minMaxZoom[1]);
 }
 
+/// Converts platform interface's JointType to Pigeon's PlatformJointType.
+@visibleForTesting
+PlatformJointType platformJointTypeFromJointType(JointType jointType) {
+  switch (jointType) {
+    case JointType.mitered:
+      return PlatformJointType.mitered;
+    case JointType.bevel:
+      return PlatformJointType.bevel;
+    case JointType.round:
+      return PlatformJointType.round;
+  }
+  // The JointType enum is defined in the platform interface package. A fallback return value here ensures the android package will not break if a new enum value is added.
```

Comment:
Please wrap comments to the same line length as the code.

Although I would prefer if this used the exact text that I linked to. We've used it in many other places, so it would be more consistent, and it provides more detail about the reason for the specific structure so that someone doesn't come along and think it would be cleaner to fold this into a `default` case.